roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -1,107 +1,401 @@
+- `Seconded` statements must be sent before `Valid` statements.
+- `Seconded` statements may only be sent to other members of the group when the
+  candidate is fully known by the local validator.
+  - "Fully known" means the validator has the full `CommittedCandidateReceipt`
+    and `PersistedValidationData`, which it receives on request from other
+    validators or from a collator.
+  - The reason for this is that sending a statement (which is always a
+    `CompactStatement` carrying nothing but a hash and signature) to the
+    cluster, is also signal that you are available to request the candidate
+    from.
+  - This makes the protocol easier to reason about, while also reducing network
+    messages about candidates that don't really exist.
+- Validators in a cluster receiving messages about unknown candidates request
+  the candidate (and statements) from other cluster members which have it.
+- Spam considerations
+  - The maximum depth of candidates allowed in asynchronous backing determines
+    the maximum amount of `Seconded` statements originating from a validator V
+    which each validator in a cluster may send to others. This bounds the number
+    of candidates.
+  - There is a small number of validators in each group, which further limits
+    the amount of candidates.
+- We accept candidates which don't fit in the fragment trees of any relay
+  parents.
+  - We listen to prospective parachains subsystem to learn of new additions to
+    the fragment trees.
+  - Use this to attempt to import the candidate later.
+
+### Grid Mode
+
+- Every consensus session provides randomness and a fixed validator set, which
+  is used to build a redundant grid topology.
+  - It's redundant in the sense that there are 2 paths from every node to every
+    other node.
+- This grid topology is used to create "sending" and "receiving" paths from each
+  validator group to every validator.
+- When a node observes a candidate as backed, it sends a
+  `BackedCandidateManifest` to their "receiving" nodes.
+- If receiving nodes don't yet know the candidate, they request it.
+- Once they know the candidate, they respond with a
+  `BackedCandidateAcknowledgement`.
+- Once two nodes perform a manifest/acknowledgement exchange, they can send
+  `Statement` messages directly to each other for any new statements they might
+  need.
+  - This limits the amount of statements we'd have to deal with w.r.t.
+    candidates that don't really exist. See "Manifest Exchange" section.
+- There are limitations on the number of candidates that can be advertised by
+  each peer, similar to those in the cluster. Validators do not request
+  candidates which exceed these limitations.
+- Validators request candidates as soon as they are advertised, but do not
+  import the statements until the candidate is part of the hypothetical
+  frontier, and do not re-advertise or acknowledge until the candidate is
+  considered both backed and part of the hypothetical frontier.
```

Comment:
I had some difficulty with this as well in the beginning, thanks for pointing it out. 👍 I'm not sure if this needs to be fixed or if this usage of "backed" is an accepted shorthand in this context.

 @rphmeier bringing this to your attention. Maybe it can be addressed in the refactor.

Reply:
This would bring way too much refactoring, backed and backable are interchangeable, it's easy enough to figure it from the context

Reply:
I suppose I'd agree that the distinction between backed and backable doesn't impact much here. It just required a bit of extra thinking on my part to reason that backed in this context doesn't also mean posted on-chain. 

In the provisioner subsystem the distinction matters more. Good point that the refactor could be prohibitively large, since these terms get included in names across most subsystems.

I would like to point out that for us relatively new folks it's much harder to tell which meaning of a term is intended if there are multiple.

Reply:
I agree that this is a source of confusion (certainly has been for me). It should be something to keep in mind for the future, at least. 

Concerning the refactor, I don't think I meant that we should go back and rename all the enum variants etc. since that would be a huge change, but just to make the meaning more clear where it makes sense to do so. Sorry for not being clear at all about that, I was pretty sick and out of it haha.

Reply:
I don't think different words with different meanings should be used to describe the same thing. Somehow defeats the purpose of having different words with different meanings, imho